packages/x-license-pro/src/utils/licenseErrorMessageUtils.ts

```diff
@@ -20,6 +20,17 @@ export function showInvalidLicenseError() {
   ]);
 }
 
+export function showOutOfScopeLicenseError() {
+  showError([
+    'MUI: Out of scope license key.',
+    '',
+    'Your MUI X license key isn\'t valid. You are rendering a DataGridPremium component that requires a license key with the "premium" feature scope but your license key has the "pro" feature scope.',
+    '',
+    'You can solve the issue by purchasing a premium license at https://mui.com/r/x-get-license?scope=premium',
```

Comment:
It seems that we use Pro and Premium as a noun.

```suggestion
    'You can solve the issue by purchasing a Premium license at https://mui.com/r/x-get-license?scope=premium',
```